__tests__/home/home.test.js

```diff
@@ -155,6 +187,45 @@ describe('Home Page', () => {
     expect(latestSyncStatusText).not.toBe(`Last Sync: In progress`);
   });
 
+  it('should display the Sync Idle 7d+ Users On Discord button', async () => {
+    const syncIdle7dPlusUsersButton = await page.$('#sync-idle-7d-Plus-users');
+    expect(syncIdle7dPlusUsersButton).toBeTruthy();
+
+    const spinnerInsideSyncIdle7dPlusUsers = await syncIdle7dPlusUsersButton.$(
+      '.spinner',
+    );
+    expect(spinnerInsideSyncIdle7dPlusUsers).toBeTruthy();
+
+    const syncIdle7dPlusUsersUpdate = await page.$(
+      '#sync-idle-7d-Plus-users-update',
+    );
+    expect(syncIdle7dPlusUsersUpdate).toBeTruthy();
+  });
+
+  it('should display the latest sync date when a super_user clicks on the Sync Idle 7d+ Users On Discord button', async () => {
+    await page.evaluate(() => {
+      document.querySelector('#sync-idle-7d-Plus-users').click();
+    });
+    await page.waitForNetworkIdle();
+
+    const latestSyncStatusElement = await page.waitForSelector(
+      '#sync-idle-7d-Plus-users-update',
+    );
+
+    expect(latestSyncStatusElement).toBeTruthy();
+
+    const latestSyncStatusText = await page.evaluate(
+      (element) => element.textContent,
+      latestSyncStatusElement,
+    );
+
+    expect(latestSyncStatusText).not.toBe(`Last Sync: Failed`);
+    expect(latestSyncStatusText).not.toBe(
+      `Last Sync: Synced Data Not Available`,
+    );
+    expect(latestSyncStatusText).not.toBe(`Last Sync: In progress`);
+  });
```

Comment:
NIT: is it possible to test the exact message that will be displayed on screen after idle user sync?